docs/azure-config.md

```diff
@@ -0,0 +1,167 @@
+* Create an AWS IAM user for Ark
+* Configure the server
+* Create a Secret for your credentials
+
+If you do not have the `az` Azure CLI 2.0 locally installed, follow the [install guide][18] to set it up. Once done, run:
+
+```bash
+az login
+```
+
+## Kubernetes cluster prerequisites
+
+Ensure that the VMs for your agent pool allow Managed Disks. If I/O performance is critical,
+consider using Premium Managed Disks, which are SSD backed.
+
+## Create Azure storage account and blob container
+
+Heptio Ark requires a storage account and blob container in which to store backups.
+
+The storage account can be created in the same Resource Group as your Kubernetes cluster or
+separated into its own Resource Group. The example below shows the storage account created in a
+separate `Ark_Backups` Resource Group.
+
+The storage account needs to be created with a globally unique id since this is used for dns. The
+random function ensures you don't have to come up with a unique name. The storage account is
+created with encryption at rest capabilities (Microsoft managed keys) and is configured to only
+allow access via https.
+
+```bash
+# Create a resource group for the backups storage account. Change the location as needed.
+AZURE_BACKUP_RESOURCE_GROUP=Ark_Backups
+az group create -n $AZURE_BACKUP_RESOURCE_GROUP --location WestUS
+
+# Create the storage account
+AZURE_STORAGE_ACCOUNT_ID="ark`cat /proc/sys/kernel/random/uuid | cut -d '-' -f5`"
+az storage account create \
+    --name $AZURE_STORAGE_ACCOUNT_ID \
+    --resource-group $AZURE_BACKUP_RESOURCE_GROUP \
+    --sku Standard_GRS \
+    --encryption-services blob \
+    --https-only true \
+    --kind BlobStorage \
+    --access-tier Hot
+
+# Create the blob container named "ark". Feel free to use a different name; you'll need to
+# adjust the `bucket` field under `backupStorageProvider` in the Ark Config accordingly if you do.
+az storage container create -n ark --public-access off --account-name $AZURE_STORAGE_ACCOUNT_ID
+
+# Obtain the storage access key for the storage account just created
+AZURE_STORAGE_KEY=`az storage account keys list \
+    --account-name $AZURE_STORAGE_ACCOUNT_ID \
+    --resource-group $AZURE_BACKUP_RESOURCE_GROUP \
+    --query [0].value \
+    -o tsv`
+```
+
+## Create service principal
+To integrate Heptio Ark with Azure, you should follow the instructions below to create an Ark-specific [service principal][17].
+
+2. There are seven environment variables that need to be set for Heptio Ark to work properly. The following steps detail how to acquire these, in the process of setting up the necessary RBAC.
+
+3. Obtain your Azure Account Subscription ID and Tenant ID:
+
+    ```bash
+    AZURE_SUBSCRIPTION_ID=`az account list --query '[?isDefault].id' -o tsv`
+    AZURE_TENANT_ID=`az account list --query '[?isDefault].tenantId' -o tsv`
+    ```
+
+4. Set the name of the Resource Group that contains your Kubernetes cluster.
```

Comment:
Would be nice to cover https://github.com/heptio/ark/issues/306 here

Reply:
Made an attempt, but short on time to test and feel confident of wording.